doc/nrf/releases/release-notes-1.8.0.rst

```diff
@@ -715,19 +713,19 @@ Zephyr
 
 .. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
 
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``14f09a3b00``, plus some |NCS| specific additions.
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``3f82656``, plus some |NCS| specific additions.
 
 For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
 
 .. code-block:: none
 
-   git log --oneline 14f09a3b00 ^v2.6.0-rc1-ncs1
+   git log --oneline 3f82656 ^v2.7.0
 
 For a complete list of |NCS| specific commits, run:
 
 .. code-block:: none
 
-   git log --oneline manifest-rev ^14f09a3b00
+   git log --oneline manifest-rev ^3f82656
```

Comment:
@nvlsianpu , please double-check.

Reply:
- fixed support for low power in zephyr's single thread mode, see MCUBOOT_CPU_IDLE() macro
- switched usb cdc-acm serial recovery I/O device configuration from Kconfig to DT with compatible = "zephyr,cdc-acm-uart"
- switched uart serial recovery I/O device configuration from Kconfig to DT using zephyr's zephyr,console property of the chosen node.
- fixed deadlock on cryptolib selectors in Kconfig
- Fixed serial recovery skipping on nrf5340
- Added cleanup of uarte devices before chainload the application which allows the application to initialize them correctly (this fixes for instance possible missing output log issue)

Reply:
this is ok